### PR TITLE
MM-35053 - cloud trial banner fixes

### PR DESCRIPTION
**Pull request by pablovelezvidal**
#### Summary
Fix the styles for the banners so the text and the button look more aligned

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-35053

#### Screenshots
Captura de pantalla 2021-04-22 a las 0 22 36
Captura de pantalla 2021-04-22 a las 0 23 03
Captura de pantalla 2021-04-22 a las 0 23 27

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -264,7 +264,7 @@
   "admin.billing.subscription.cancelSubscriptionSection.description": "At this time, deleting a workspace can only be done with the help of a customer support representative.",
   "admin.billing.subscription.cancelSubscriptionSection.title": "Cancel your subscription",
   "admin.billing.subscription.cloudTrial.daysLeftOnTrial": "There are {daysLeftOnTrial} days left on your free trial",
-  "admin.billing.subscription.cloudTrial.lastDay": "This is the last day of your free trial. Your access will expire at 11:59pm PT.",
+  "admin.billing.subscription.cloudTrial.lastDay": "This is the last day of your free trial. Your access will expire today at 11:59pm PT.",
```
Comment (nickmisasi) [inline i18n/en.json:267]: Now that I think about it, the 11:59 is UTC not PT, so we should probably change this. 2 suggestions:

1. We display the time in UTC
2. We parse the time into the local time and display that

**Review by nickmisasi — APPROVED** [review]

**Review by annelieseklein — APPROVED** [review]
looks great!

File: i18n/en.json (hunk 1 above)
Comment (pablovelezvidal) [reply i18n/en.json:267]: @marianunez  @annelieseklein thoughts around Nick suggestions?

**Comment by annelieseklein** [comment]
@pablovelezvidal @nickmisasi can we just display whichever timezone the user is in instead of UTC?

File: i18n/en.json (hunk 1 above)
Comment (marianunez) [reply i18n/en.json:267]: +1 to show the time in user's local time

Comment (pablovelezvidal) [reply i18n/en.json:267]: +1 to do that in another ticket.

Comment (marianunez) [reply i18n/en.json:267]: @pablovelezvidal can you create the ticket to keep track of it outside of this PR?

**Review by marianunez — APPROVED** [review]

**Comment by marianunez** [comment]
@stevemudie could we test this post merge? @pablovelezvidal if it's good with you, I propose to merge so we can include in the cloud Fast forward today.

**Comment by pablovelezvidal** [comment]
> @stevemudie could we test this post merge? @pablovelezvidal if it's good with you, I propose to merge so we can include in the cloud Fast forward today.

yes, no problem, let me merge it then.

File: i18n/en.json (hunk 1 above)
Comment (marianunez) [reply i18n/en.json:267]: @pablovelezvidal reminder on this so we don't forget

Comment (pablovelezvidal) [reply i18n/en.json:267]: @marianunez https://mattermost.atlassian.net/browse/MM-35191

**Pull request merged**